src/integrationTest/java/org/radarcns/integration/util/ExpectedDocumentFactory.java

```diff
@@ -50,28 +50,36 @@
      * @param statistic function that has to be returned
      * @param collectors array of aggregated data
      * @return the set of values that has to be stored within a {@code Dataset} {@code Item}
-     * @see {@link DoubleValueCollector}
+     * @see DoubleValueCollector
      **/
-    public List<? extends Object> getStatValue(StatType statistic,
+    public List<?> getStatValue(StatType statistic,
             DoubleArrayCollector collectors) {
 
         switch (statistic) {
             case AVERAGE:
-                return collectors.convertToAvro().getAvg();
+                return collectors.getCollectors().stream()
+                        .map(DoubleValueCollector::getAvg).collect(Collectors.toList());
```

Comment:
Yes, just because `convertToAvro` is no longer available... But I see in the dev branch you already implemented a List summation. I can remove this and use your code instead.